### PR TITLE
Add documentation for some gen.py functions

### PR DESCRIPTION
**Pull request by dalepotter**

### CODE REVIEW

File: gen.py [hunk 1]
```diff
@@ -26,15 +35,28 @@ def get_github_url(repo, path=''):
...
+        str: The human-friendly represention of the list.
     """
     return ', '.join(l)
 
-
 def lookup_see_also(standard, mapping, path):
+    """Return a generator object containing ???
+
+    Args:
+        standard (str): Unknown
```
Comment (allthatilk) [inline gen.py]: standard (str): 'activity-standard' presumably a tag indicating that it is working from the Activity Standard

File: gen.py [hunk 2]
```diff
@@ -26,15 +35,28 @@ def get_github_url(repo, path=''):
...
+        mapping (list): Unknown
```
Comment (allthatilk) [inline gen.py]: mapping (dict): A dictionary containing tuples in the format key:[value] that groups Element tags in the Standard by relevant category. eg. `u'activity-file': [u'iati-activities', u'iati-activities/iati-activity']`

File: gen.py [hunk 3]
```diff
@@ -26,15 +35,28 @@ def get_github_url(repo, path=''):
...
+        path (str): Unknown
```
Comment (allthatilk) [inline gen.py]: path (str): Seems to return strings for xpath queries. eg. `'other-identifier'` or `'activity-date/narrative'`

File: gen.py [hunk 4]
```diff
@@ -26,15 +35,28 @@ def get_github_url(repo, path=''):
...
+
+    Returns:
+        generator or str: Unknown
```
Comment (allthatilk) [inline gen.py]: generator (str): Generates a series of unicode strings for Activity Elements in Schema (I think).

File: gen.py [hunk 5]
```diff
@@ -298,15 +330,22 @@ def output_overview_page(self, standard, page, reference_pages):
...
-        """
-        Loop over the children of a given element, and run output_docs on each.
-
-        Returns the names of the child elements.
-
+        """Find child elements for a given input element.
+
+        Args:
+            element (lxml.etree._Element): The base element to find child elements for.
+            path (str): Unused.
```
Comment (allthatilk) [inline gen.py:366]: If it's unused is it okay to remove the arg?

**Review by allthatilk — CHANGES_REQUESTED** [review]

**Review by samuele-mattiuzzo — APPROVED** [review]

**Pull request merged**